app/javascript/components/world-map/WorldMap.js

```diff
@@ -20,6 +21,26 @@ const PetersGall = geoCylindricalEqualArea().parallel(45);
 function WorldMap(props) {
   const { layers } = props;
 
+  const getTooltip = (tooltipContent) => {
+    const { country, climateLawsPolicies, emissions, link } = tooltipContent;
+    return (
+      <div className="world-map__tooltip-container">
+        <p className="world-map__tooltip-title">{country}</p>
+        <div className="world-map__tooltip-row">
+          <p className="world-map__tooltip-text">Number of climate laws and policies</p>
+          <p className="world-map__tooltip-number">{climateLawsPolicies}</p>
+        </div>
+        <div className="world-map__tooltip-row">
+          <p className="world-map__tooltip-text">Greenhouse gas emissions (% of global emissions)</p>
+          <p className="world-map__tooltip-number">{emissions}</p>
+        </div>
+        <a href="https://todo-link-to-that-country" className="world-map__tooltip-link">{link}</a>
+      </div>
+    );
+  };
+
+  const [content, setTooltipContent] = useState('');
```

Comment:
I think I will move that into that reducer, we should either use reducer or state I guess?